src/modules/emote_menu/twitch/EmoteMenu.module.less

```diff
@@ -20,4 +20,4 @@
     background-color: var(--color-background-button-text-hover);
     color: var(--color-fill-button-icon-hover);
   }
-}
+}
```

Comment:
can we keep this newline, and why was this and the other file swapped from postcss to less?